javatests/arcs/android/host/parcelables/ParcelableParticleSpecTest.kt

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+package arcs.android.host.parcelables
+
+import android.os.Parcel
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import arcs.core.common.ArcId
+import arcs.core.data.EntityType
+import arcs.core.data.FieldType.Companion.Text
+import arcs.core.data.HandleConnectionSpec
+import arcs.core.data.ParticleSpec
+import arcs.core.data.Schema
+import arcs.core.data.SchemaDescription
+import arcs.core.data.SchemaFields
+import arcs.core.data.SchemaName
+import arcs.core.storage.driver.VolatileStorageKey
+import arcs.core.type.TypeFactory
+import com.google.common.truth.Truth.assertThat
+import org.junit.Test
+import org.junit.runner.RunWith
+
+/** Tests for [ParcelableParticleSpec]'s classes. */
+@RunWith(AndroidJUnit4::class)
+class ParcelableParticleSpecTest {
+
+    @Test
+    fun particleSpec_parcelableRoundTrip_works() {
+        val personSchema = Schema(
+            listOf(SchemaName("Person")),
+            SchemaFields(mapOf("name" to Text), emptyMap()),
+            SchemaDescription(),
+            "42"
+        )
+
+        val connectionSpec = HandleConnectionSpec(
+            VolatileStorageKey(
+                ArcId.newForTest("foo"), "bar"
```

Comment:
Either put `"bar"` on the next line, or make the whole VolatileStorageKey sit on one line. (should fit within 100 columns no problem)